README.md

```diff
@@ -77,13 +75,20 @@ Build the library and demo program.
 $ make
 ```
 
-Run sample `Mado` program:
+Run sample `Mado` program with SDL backend:
 ```shell
 $ ./demo-sdl
 ```
 
 Once the window appears, you should be able to move the windows and interact with the widgets.
 
+Run sample `Mado` program with the Linux framebuffer backend:
+```shell
+$ sudo FRAMEBUFFER=/dev/fb0 MOUSE=/dev/input/mice ./demo-fbdev
```

Comment:
Follow the section "Environment" and "Run the demo application" of [lv_port_linux](https://github.com/lvgl/lv_port_linux).

Reply:
Don't mix `sudo` and `FRAMEBUFFER` in a single command. Make it explicitly clear.

Reply:
`sudo` is not necessary if the user is in the `video` group.